packages/winston-log-enricher/lib/createFormatter.js

```diff
@@ -50,11 +50,17 @@ module.exports = function createFormatter(newrelic, winston) {
       info[m] = metadata[m]
     })
 
-    const levelLabel = info.level
-    newrelic.shim.agent.metrics.getOrCreateMetric('Logging/lines').incrementCallCount()
-    newrelic.shim.agent.metrics
-      .getOrCreateMetric(`Logging/lines/${levelLabel}`)
-      .incrementCallCount()
+    const config = newrelic.shim.agent.config
+
+    // TODO: update the peerdep on the New Relic repo and thereby
+    // remove check for existence of application_logging config item
+    if (config.application_logging && config.application_logging.metrics.enabled) {
```

Comment:
so after looking at the spec and seeing comments from my questions this is slightly wrong.  both `config.application_logging.enabled` and `config.application_logging.metrics.enabled` need to be true before logging these metrics.  I'll open a PR to fix this